### DISCUSSION TITLE
WEEK 1 WITH HELL DIFFICULTY + TUTORIAL

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#24] [Source Show & Tell]
YEEESSS I MADE HELL FOR WEEK 1 AND TUTORIAL WEEEEEE

**Comment by ThatOneSillyGuyDon** [comment]
Oh god, they better be playable, cause I’m still charting all of them

**Reply by Theoyeah** [reply]
There are playable + In tutorial there still the funni weird end but harder :trollface: (still fc)
